chronos/1.10/README.md

```diff
@@ -0,0 +1,128 @@
+chronos  2.5.0    /chronos  ---      A fault tolerant job scheduler for Mesos which handles...
+```
+
+## Create a scheduled job
+
+Open the Chronos UI from the DC/OS UI via the `Open Service` button and click the `New Job` link to bring up the form for creating jobs.
+
+![New job form in Chronos](img/new-job.png)
+
+Next, fill in the following values in the form:
+
+- `NAME`: "Date"
+- `SCHEDULE`: Enter `T10S` in the `P` field
+- `COMMAND`: `/bin/date`
+
+
+Note that Chronos uses [ISO 8601 Interval Notation](https://en.wikipedia.org/wiki/ISO_8601#Time_intervals) to describe job schedules, so `T10S` means run this job every 10 seconds.
+
+Now click the `Create` button at the top of the form to submit the job.
+
+Let's verify that our job ran successfully. Run the following CLI command to view all completed tasks:
+
+```bash
+$ dcos task --completed ct*
+NAME              HOST            USER  STATE  ID                        MESOS ID                                 
+ChronosTask:Date  192.168.65.111  root    F    ct:1509055284000:0:Date:  f99ad98b-cc45-42c8-b7a2-d4c96ade99f4-S2  
+ChronosTask:Date  192.168.65.111  root    F    ct:1509055344000:0:Date:  f99ad98b-cc45-42c8-b7a2-d4c96ade99f4-S2  
+ChronosTask:Date  192.168.65.111  root    F    ct:1509055354000:0:Date:  f99ad98b-cc45-42c8-b7a2-d4c96ade99f4-S2  
+ChronosTask:Date  192.168.65.111  root    F    ct:1509055364000:0:Date:  f99ad98b-cc45-42c8-b7a2-d4c96ade99f4-S2  
+ChronosTask:Date  192.168.65.111  root    F    ct:1509055374000:0:Date:  f99ad98b-cc45-42c8-b7a2-d4c96ade99f4-S2  
+ChronosTask:Date  192.168.65.111  root    F    ct:1509055384000:0:Date:  f99ad98b-cc45-42c8-b7a2-d4c96ade99f4-S2  
+ChronosTask:Date  192.168.65.111  root    F    ct:1509055394000:0:Date:  f99ad98b-cc45-42c8-b7a2-d4c96ade99f4-S2  
+ChronosTask:Date  192.168.65.111  root    F    ct:1509055404000:0:Date:  f99ad98b-cc45-42c8-b7a2-d4c96ade99f4-S2  
+ChronosTask:Date  192.168.65.111  root    F    ct:1509055414000:0:Date:  f99ad98b-cc45-42c8-b7a2-d4c96ade99f4-S2  
+```
+Note that the `--completed` argument includes tasks that have completed their execution. Chronos uses the prefix `ct` for all its tasks, so `ct*` filters out Chronos tasks.
+
+To view the output of a task, copy one of the values under the `ID` column in the output of the previous command and use it as the argument to `dcos task log`:
+
+```bash
+$ dcos task log --completed ct:1509055574000:0:Date:  
+Executing pre-exec command '{"arguments":["mesos-containerizer","mount","--help=false","--operation=make-rslave","--path=\/"],"shell":false,"value":"\/opt\/mesosphere\/active\/mesos\/libexec\/mesos\/mesos-containerizer"}'
+Executing pre-exec command '{"shell":true,"value":"mount -n -t proc proc \/proc -o nosuid,noexec,nodev"}'
+```
+
+You can also see the status of the job you entered in the Chronos UI:
+
+![List of jobs in Chronos](img/status.png)
+
+Or in the Services > chronos log output in DC/OS UI:
+
+![DC/OS Service Tasks Logs](img/output.png)
```

Comment:
This is a screenshot of the STDERR log, which doesn't show the results of the job, and shows failures. Can you instead take a screenshot of the STDOUT log, like we have used in the examples from previous versions?